scripts/metrics-config.yml

```diff
@@ -0,0 +1,65 @@
+"cdc/log.cc":
```

Comment:
This file will possibly get out-of-date. Who will maintain it?

Reply:
The short truth is,  I will. Currently, on every release, I'm collecting metrics by running Scylla. It needs to find all the metrics but cannot.
That was the original idea behind #16328, which faced a few fundamental issues and was closed.

The end goal is for the script to run as part of the compilation; it will fail if someone tries to do something funky or will make an error, like having different descriptions for the same metrics.

I've already opened multiple issues around subjects I identified with the script; I'm counting on solving as many issues as possible; in a perfect world, everything will be in the code.
But it will take time. Meanwhile, for the first time, I got a complete list of our metrics and it's a good step forward.